packages/nimble-components/src/wafer-map/types.ts

```diff
@@ -20,6 +20,14 @@ export const WaferMapOrientation = {
 export type WaferMapOrientation =
     (typeof WaferMapOrientation)[keyof typeof WaferMapOrientation];
 
+export const HoverDieOpacity = {
```

Comment:
Could you add a test to `types.spec.ts` for this new type, matching the pattern of the other enum-like types there?